tests/scripts/all.sh

```diff
@@ -2117,7 +2118,7 @@ component_test_memsan_constant_flow_psa () {
 }
 
 component_test_valgrind_constant_flow () {
-    check_test_extended || return
+    check_test_full || return
```

Comment:
Not sure if we can do this within `all.sh`, but could we have an alias `only_on_full_test_run`?

```
alias only_on_full_test_run="check_test_full || return"                                                                                                          
```

Then we'd just do

```
component_test_valgrind_constant_flow () {
    only_on_full_test_run
    :
}
```

Would that be clearer, and less prone to error?

Reply:
I wondered about this but didn't know the right way to express it in bash. TBH I think this PR is already a little bit hacky; probably better to keep it simple.